src/main/java/us/warframestat/moddetection/api/detection/DetectModInfo.java

```diff
@@ -47,13 +49,12 @@ public static void setup() {
     tesseract.setLanguage("eng");
     tesseract.setTessVariable("user_defined_dpi", "70");
 
-    Path dataDirectory;
-    try {
-      dataDirectory = Paths.get(ClassLoader.getSystemResource("tesseract").toURI());
-    } catch (URISyntaxException e) {
+    Path tesseractPath = API.DATA.toPath().resolve("tesseract");
+    if (!tesseractPath.toFile().exists()) {
+      logger.info("{\"error\": \"tesseract folder not found\"}");
```

Comment:
In the future, let's look at preconstruction error JSON/GSON objects so we don't do literals that might get problematic later